superset/views/annotations.py

```diff
@@ -43,7 +43,7 @@ def __call__(self, form, field):
             )
 
 
-class AnnotationModelView(SupersetModelView, DeleteMixin):  # noqa
+class AnnotationModelView(SupersetModelView, DeleteMixin):
```

Comment:
Out of scope, but shouldn't mixins in general be before the base class? Seems like an antipattern to expect the base class to have functionality that overrides the mixin..

Reply:
Valid.